pallets/phala/src/mining.rs

```diff
@@ -625,6 +630,7 @@ pub mod pallet {
 							Some(miner) => miner,
 							None => continue, // Skip non-existing miners
 						};
+						// The MiningActive is removed, is this logic need to be changed?
 						// Skip non-mining miners
 						if !miner_info.state.is_mining() {
```

Comment:
In the previous, `is_mining()` meant MiningIdle or MiningUnresponsive (MiningActive was excluded). Now it means MiningIdle (including previous MiningActive), MiningUnresponsive.

Reply:
No problem. We never have `Active` state in the past